persistent/Database/Persist/Sql/Types.hs

```diff
@@ -36,9 +36,13 @@ data Column = Column
 -- @since 2.11.0.0
 data ColumnReference = ColumnReference
     { crTableName :: !EntityNameDB
-    -- ^ The table name that the
+    -- ^ The foreign table's name.
```

Comment:
It's not necessarily foreign right? You can have a self-referential FK. I guess it's not totally clear by what's meant by foreign in this context